tests/fixtures/apply_rules/output_consolidation_with_object_code/physical_modes.txt

```diff
@@ -0,0 +1,4 @@
+physical_mode_id,physical_mode_name,co2_emission
+Bus,Bus,130.5
+Metro,Metro,90.0
+RapidTransit,Rapid Transit,
```

Comment:
I am not sure of the need for this file. There is no use of the property_rules.txt in the test.
The PR is ok otherwise.